bitnami/harbor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: harbor
-version: 4.0.2
+version: 4.1.0
```

Comment:
This is just a fix, right? I would increase patch version in that case

Reply:
I'm not sure, in fact, we are changing the required content in the secret, it is that a fix or adding/modifying a feature. I don't really know, yes I would go for a patch. Changing it